optimum_benchmark/experiment.py

```diff
@@ -114,13 +113,11 @@ class ExperimentConfig:
 
 @hydra.main(version_base=None)
 def run_experiment(experiment: DictConfig) -> None:
-    # By default, Hydra populates the metadata object_type with the ones from ExperimentConfig but the object_type should really be
-    # one of the subclass (e.g. PyTorchBackendConfig instead of BackendConfig). This is required to call `to_object`.
-    experiment = remap_to_correct_metadata(experiment)
+    from omegaconf import SCMode
 
-    # This is required to trigger __post_init__. Reference: https://github.com/omry/omegaconf/issues/377
```

Comment:
I think this comment is useful to know why this code is there.

Reply:
true, I put it back.